piper/CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # Changelog
 
+## 0.1.3
+
+- Fix multi-line input
+- Verify voice hashes on download
```

Comment:
Usually we link to the changelog of the upstream project (e.g. in Z-Wave JS to the Z-Wave JS tag). Maybe we can do the same?

Reply:
Looks like there is no such thing

Reply:
Yeah, sorry. The code for the library is buried in a branch of the Rhasspy 3 repo.